ImperatorToCK3Tests/ImperatorWorldTests/Provinces/PopTests.cpp

```diff
@@ -1,17 +1,20 @@
 #include "gtest/gtest.h"
 #include "../ImperatorToCK3/Source/Imperator/Provinces/Pop.h"
+#include "../ImperatorToCK3/Source/Imperator/Provinces/PopFactory.h"
 #include <sstream>
 
+
+Imperator::Pop::Factory popFactory;
```

Comment:
Either use a test fixture to own the popFactory, or do Imperator::Pop::Factory{}.getPop() in the tests to create a factory. Don't create at global scope.

Reply:
Used `Imperator::Pop::Factory{}.getPop()` ✔️